### PR TITLE
support .env file for environment variables.

### PR DESCRIPTION
**Pull request by HRanjan-11**

### CODE REVIEW

File: commands/utils/set_args.js [hunk 1]
```diff
@@ -385,6 +386,36 @@ function sync_args_from_cmd(args) {
         }
       }
     }
+    let dot_env_vars = undefined;
+    if ("sys-env-keys" in args) {
+      dot_env_vars = args["sys-env-keys"];
+    } else if (lt_config["run_settings"]["sys_env_keys"]) {
```
Comment (shubhamlt) [inline commands/utils/set_args.js:392]: can we check it like lt_config["run_settings"] && lt_config["run_settings"]["sys_env_keys"]

Comment (HRanjan-11) [reply commands/utils/set_args.js]: done

File: commands/utils/set_args.js [hunk 2]
```diff
@@ -385,6 +386,36 @@ function sync_args_from_cmd(args) {
...
+        envFilePath = args["envfl"];
+      } else if (lt_config["run_settings"]["env_file"]) {
+        envFilePath = lt_config["run_settings"]["env_file"];
+      }
+      
+      try {
+        envFile = fs.readFileSync(envFilePath, {encoding: 'utf8'})
+        parsedEnv = dotenv.parse(envFile)
+        for (index in dot_env_vars) {
+          envKey = dot_env_vars[index]
```
Comment (amathur-LT) [inline commands/utils/set_args.js:410]: envKey and envValue  should be declared as local var with `let`

**Review by amathur-LT — CHANGES_REQUESTED** [review]

File: commands/utils/set_args.js (hunk 2 above)
Comment (HRanjan-11) [reply commands/utils/set_args.js]: done

**Pull request merged**